ggj25/scenes/Bubball_HUD.tscn

```diff
@@ -124,7 +124,6 @@ theme_override_font_sizes/font_size = 32
 text = "Controls"
 
 [node name="Settings" type="Button" parent="MarginContainer2/GamePauseButtons"]
-visible = false
```

Comment:
What does this change?

Reply:
This makes the settings button show on the pause menu

Reply:
Ah. So that button, I don't think its ready because it opens a new scene. If you go back to the game after settings, it will create a new game

Reply:
Ah ofc - I will make a new issue